client/app/mobile/routes.js

```diff
@@ -1,9 +1,14 @@
+import React from 'react';
 import { StackNavigator } from 'react-navigation';
 
-import MasterSeries from './modules/series/masterSeries';
+import MasterSeriesSF from '../modules/series/root/MasterSeriesSF';
```

Comment:
It seems strange

Reply:
No, this is because was changed to use the HOC to shared component funcionality. Mobile side has not been develope/modify yet